install

```diff
@@ -2,20 +2,26 @@
 
 cd `dirname $0`
 
-# The clean script removes temporary and junk files that should not be
-# committed to the source code repo.
-if (test -e clean)
-then
-    ./clean
-fi
-
 if [ "$1" = "--skip-types" ]; then
     shift;
     dont_skip_types=0
 else
     dont_skip_types=1
 fi
 
+dev_install=0
+if [ "$1" == "--dev" ]; then
+    shift;
+    dev_install=1
+fi
+
+# The clean script removes temporary and junk files that should not be
+# committed to the source code repo.
+if [ $dev_install = 0 ]; then
```

Comment:
Shouldn't there be 2 equals signs? (same goes for lines 5, 48, 59 & 68)

Reply:
Thanks to bash and backwards compatibility, [either one is OK](https://tldp.org/LDP/abs/html/comparison-ops.html).